api/src/main/java/org/pmiops/workbench/actionaudit/auditors/FreeTierAuditor.kt

```diff
@@ -0,0 +1,5 @@
+package org.pmiops.workbench.actionaudit.auditors
+
+interface FreeTierAuditor {
```

Comment:
This was a fun exercise for Baby's First Kotlin, but it might be more appropriately part of User or Profile Auditor

Reply:
sounds good